packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -93,7 +93,10 @@ export function useGridCell<T, C extends GridCollection<T>>(props: GridCellProps
     isVirtualized,
     focus,
     shouldSelectOnPressUp,
-    onAction: onCellAction ? () => onCellAction(node.key) : onAction
+    onAction: onCellAction ? () => onCellAction(node.key) : onAction,
+    // TODO: this will affect all grids, maybe make it specific to Table via useTableState?
+    // something like isInteractionDisabled: state.collection.size === 0
+    isDisabled: state.collection.size === 0
```

Comment:
is it both non-interactive and non-navigable? or should we name the prop something else? or is that what you're suggesting in the comment?
Lets only do this for Table for now? I assume we'll be asked to do this for ListView eventually, but sounds like not yet?

Reply:
The PR also covers ListView.

Also, if the collection is empty, wouldn't there be no grid cells? I guess only in the case of tableview because of the headers?

Reply:
Yep, only TableView has grid cells when the collection is empty, CardView and TagGroup don't. Was just a bit worried that there might people using these grid hooks directly to build a table (or some other grid pattern) that wouldn't want to disable interactions when their table is empty